tests/test_trimming.py

```diff
@@ -1388,9 +1465,16 @@ def test_TrimPortStats(self, portData):
             sai_qualifiers=sai_attr_dict
         )
 
-    def test_TrimQueueStats(self, queueData):
+    @pytest.mark.parametrize(
+        "attr, value", [
+            pytest.param("SAI_QUEUE_STAT_TRIM_PACKETS", "1000", id="trim-packet"),
+            pytest.param("SAI_QUEUE_STAT_TX_TRIM_PACKETS", "2000", id="tx-packet"),
+            pytest.param("SAI_QUEUE_STAT_DROPPED_TRIM_PACKETS", "3000", id="drop-packet")
+        ]
```

Comment:
Same, clear counters for queue, port and switch

Reply:
@kperumalbfn counters clear testing is done as part of `sonic-utilities`. It has nothing to do with the VS tests of the Flex Counter infra. Please pay attention to the fact that counters operate only in a READ but not READ-and-CLEAR mode.